Assets/Scripts/Game/InputManager.cs

```diff
@@ -1081,6 +1083,16 @@ public bool AnyKeyUpIgnoreAxisBinds
             get => GetAnyKeyUpIgnoreAxisBinds() != KeyCode.None;
         }
 
+        public bool GetUiExitKeyDown()
```

Comment:
These should be moved underneath where the "GetMouseButton" functions are. These methods should also be use the word "Button" instead of "Key".